setup.py

```diff
@@ -19,6 +19,7 @@ class WebpackBuildCommand(setuptools.command.build_py.build_py):
 
     def run(self):
         if not 'CI' in os.environ:
```

Comment:
If we actually run `npm install` as it does below, perhaps we could actually run this in CI which seems somewhat useful.